src/Raven.Server/Documents/Expiration/ExpiredDocumentsCleaner.cs

```diff
@@ -119,15 +123,15 @@ private async Task DoExpirationWork()
 
         internal Task CleanupExpiredDocs(int? batchSize = null)
         {
-            return CleanupDocs(batchSize ?? BatchSize, forExpiration: true);
+            return CleanupDocs(batchSize ?? BatchSize, ExpirationConfiguration.ExpirationMaxItemsToProcess ?? MaxItemsToProcess, forExpiration: true);
         }
 
         internal Task RefreshDocs(int? batchSize = null)
         {
-            return CleanupDocs(batchSize ?? BatchSize, forExpiration: false);
+            return CleanupDocs(batchSize ?? BatchSize, RefreshConfiguration.RefreshMaxItemsToProcess ?? MaxItemsToProcess, forExpiration: false);
```

Comment:
if we add configuration from older client, will `RefreshConfiguration.RefreshMaxItemsToProcess` be null, right? I wonder if we can manually test that or create an interversion test?

Reply:
I checked it manually and it's null and working as expected :)